schema/me/conversations.js

```diff
@@ -1,112 +1,11 @@
-import date from "../fields/date"
 import impulse from "../../lib/loaders/impulse"
 import gravity from "../../lib/loaders/gravity"
-import { get } from "lodash"
 import { pageable } from "relay-cursor-paging"
+import { ConversationType } from "./conversation"
 import { parseRelayOptions } from "../../lib/helpers"
 import { connectionFromArraySlice, connectionDefinitions } from "graphql-relay"
-import { GraphQLObjectType, GraphQLString, GraphQLList, GraphQLBoolean, GraphQLNonNull, GraphQLEnumType } from "graphql"
-import { ArtworkType } from "../artwork"
 const { IMPULSE_APPLICATION_ID } = process.env
 
-export const BuyerOutcomeTypes = new GraphQLEnumType({
-  name: "BuyerOutcomeTypes",
-  values: {
-    PURCHASED: {
-      value: "purchased",
-    },
-    STILL_CONSIDERING: {
-      value: "still_considering",
-    },
-    HIGH_PRICE: {
-      value: "high_price",
-    },
-    LOST_INTEREST: {
-      value: "lost_interest",
-    },
-    WORK_UNAVAILABLE: {
-      value: "work_unavailable",
-    },
-    OTHER: {
-      value: "other",
-    },
-    BLANK: {
-      value: "",
-    },
-  },
-})
-
-export const ConversationFields = {
-  id: {
-    description: "Impulse id.",
-    type: GraphQLString,
-  },
-  inquiry_id: {
-    description: "Gravity inquiry id.",
-    type: GraphQLString,
-  },
-  from_id: {
-    type: new GraphQLNonNull(GraphQLString),
-  },
-  from_type: {
-    type: new GraphQLNonNull(GraphQLString),
-  },
-  from_name: {
-    type: new GraphQLNonNull(GraphQLString),
-  },
-  from_email: {
-    type: new GraphQLNonNull(GraphQLString),
-  },
-  to_id: {
-    type: new GraphQLNonNull(GraphQLString),
-  },
-  to_type: {
-    type: new GraphQLNonNull(GraphQLString),
-  },
-  to_name: {
-    type: new GraphQLNonNull(GraphQLString),
-  },
```

Comment:
Ohhh, I now see that these already existed previously. I think the schema could still be improved, though, if you agree with my suggestions then I think making these changes before we launch messaging is the best because no deployed clients depend on it yet (just loyalty program, which would need updating, and messaging).